src/passport-saml/strategy.ts

```diff
@@ -67,8 +65,15 @@ class Strategy extends PassportStrategy {
             throw new Error("Can't get logout response URL without a SAML provider defined.");
           }
 
-          req.samlLogoutRequest = profile;
-          return this._saml.getLogoutResponseUrl(req, options, redirectIfSuccess);
+          const samlLogoutRequest = profile;
```

Comment:
I don't know why samlLogoutRequest and profile refer to the same thing. so I kept it that way to maintain whatever the reason behind that is

Reply:
I think my question is, since you are no longer assigning `req.samlLogoutRequest`, but instead are creating a new variable via `const samlLogoutRequest`, but aren't using it, why create the variable at all?

Reply:
I updated it